OsmIntegrator.Tests/Helpers/Base/BaseHelper.cs

```diff
@@ -28,16 +29,30 @@ public async Task<HttpResponseMessage> LoginAsync(LoginData loginData)
 
         public BaseHelper(HttpClient factoryClient)
         {
-            _client = factoryClient;
+            CtorHelper(factoryClient);
```

Comment:
Marcin, co to jest? CtorHelper?

Reply:
To jest wspólna metoda wywoływana z każdego konstruktora (tu may dwa). Powinna być private a nie public. Mogę ją nazwać inaczej. Pytanie jak?

Reply:
Nie no, jest ok.